pkg/virt-api/webhooks/hyperv.go

```diff
@@ -0,0 +1,179 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2019 Red Hat, Inc.
+ *
+ */
+
+/*
+ * hyperv utilities are in the webhooks package because they are used both
+ * by validation and mutation webhooks.
+ */
+package webhooks
+
+import (
+	"fmt"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8sfield "k8s.io/apimachinery/pkg/util/validation/field"
+
+	v1 "kubevirt.io/client-go/api/v1"
+)
+
+var _true bool = true
+
+func enableFeatureState(fs **v1.FeatureState) {
```

Comment:
TIL about the `**` syntax. I think I would have passed around the vmi, but this is more elegant :)

Reply:
thanks! I'm not actually what's best readability wise between the way it is, your approach or just defining something
```golang
type FeatureStateP *v1.FeatureState

...

func enableFeatureState(fsp *FeatureStateP) {
}
```

Reply:
IMHO readability is fine as it is now. You just might need to learn about this syntax in case you don't know it yet, as I had to do, because it's not used very often. But when you know it, it makes perfectly sense.